src/coverlet.core/Properties/AssemblyInfo.cs

```diff
@@ -1,4 +1,4 @@
-[assembly: System.Reflection.AssemblyKeyFileAttribute("coverlet.core.snk")]
+[assembly: System.Reflection.AssemblyKeyFile("coverlet.core.snk")]
```

Comment:
please revert out of scope

Reply:
Reverted change, IDE made some auto fixes.